src/components/InputPhone/helpers/getCountryFromValue.ts

```diff
@@ -1,5 +1,13 @@
+import * as RNLocalize from 'react-native-localize';
 import COUNTRIES from '../countyPickerModal/countries';
 
 export default function getCountryFromValue(isoCode?: string) {
+    if (isoCode === '') {
```

Comment:
I think here we should first find `COUNTRIES.find(country => country.isoCode === isoCode)` and then if it does not exist then use the ```RNLocalize.getCountry();```
In this way if the user pass an invalid isoCode or if it is not passed at all it will fallback to the device locale